src/_pytest/doctest.py

```diff
@@ -505,7 +505,14 @@ def collect(self) -> Iterable[DoctestItem]:
         import doctest
 
         class MockAwareDocTestFinder(doctest.DocTestFinder):
-            if sys.version_info < (3, 11):
+            py_ver_info_minor = sys.version_info[:2]
+            py_ver_info_patch_segment = sys.version_info[2]
+            is_inspect_unwrap_broken = (
```

Comment:
nitpick: `inspect.unwrap` is our workaround, what's broken is doctest's `find_lineno`, so I'd go for:

```suggestion
            is_find_lineno_broken = (
```

or so.